docs/_include/mooring.rst

```diff
@@ -92,17 +94,27 @@ The lookup table dataset should contain one structure that contains fields for e
 MoorDyn
 ^^^^^^^
 
-When the MoorDyn block is used, the user needs to initiate the mooring class by 
+When a MoorDyn block is used, the user first needs to initiate the mooring class by 
 setting :code:`mooring = mooringClass('mooring name')` in the WEC-Sim input 
-file (wecSimInputFile.m), followed by the number of mooring lines that are 
-defined in MoorDyn (``mooring(1).moorDynLines = <Number of mooring lines>``) 
-
-A mooring folder that includes a moorDyn input file (``lines.txt``) is required 
-in the simulation folder. 
+file (wecSimInputFile.m), followed by setting :code:`mooring(i).moorDyn = 1` to 
+initialize a MoorDyn connection. Each MoorDyn connection can consist of multiple 
+lines and each line may have multiple nodes. The number of MoorDyn lines and nodes in 
+each line should be defined as (``mooring(i).moorDynLines = <Number of mooring lines>``) 
+and (``mooring(i).moorDynNodes(iLine) = <Number of mooring nodes in line>``), 
+respectively. 
```

Comment:
@jtgrasb Does this need to match what is defined in the MD input file? If so maybe we note that

Reply:
Yes this does - I added this to the docs.